pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2358,7 +2358,7 @@ func (c *Cloud) GetVolumeLabels(volumeName KubernetesVolumeID) (map[string]strin
 	labels := make(map[string]string)
 	az := aws.StringValue(info.AvailabilityZone)
 	if az == "" {
-		return nil, fmt.Errorf("volume did not have AZ information: %q", info.VolumeId)
+		return nil, fmt.Errorf("volume did not have AZ information: %q", aws.StringValue(info.VolumeId))
```

Comment:
`VolumeId` is a `*string`, so this seems good.

/cc @justinsb
/cc @kubernetes/sig-cloud-provider-pr-reviews

Reply:
Agreed, 
```
type Volume struct {
...
    // The ID of the volume.
    VolumeId *string `locationName:"volumeId" type:"string"`
...
}
```
from https://docs.aws.amazon.com/sdk-for-go/api/service/ec2/#Volume